pkg/runtime/depot.go

```diff
@@ -28,9 +28,10 @@ type depotConfig struct {
 }
 
 type deployment struct {
-	Type  deploymentType `json:"type"`
-	Path  string         `json:"path"`
-	Files []string       `json:"files"`
+	Type       deploymentType `json:"type"`
+	Path       string         `json:"path"`
+	Files      []string       `json:"files"`
+	InstallDir string         `json:"installDir"`
```

Comment:
Can you rename this to `RelativeSrc`? I avoided calling it `InstallDir` as I found that term to be confusing. My first assumption as to its meaning would not be the actual meaning.